extensions-core/kafka-indexing-service/pom.xml

```diff
@@ -195,10 +195,10 @@
       <artifactId>easymock</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
+    <dependency><!-- Required by dependency check, used by kafka_2.13 -->
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-library</artifactId>
-      <version>2.13.3</version>
+      <version>${scala.library.version}</version>
```

Comment:
scala is super notorious for having non-binary-compatible changes. 🤞🏼  we don't run into one of those here.